hmpps-prisoner-search/helm_deploy/values-preprod.yaml

```diff
@@ -12,9 +11,10 @@ generic-service:
   # Additional allowlist rules
   # See also allowlist values helm_deploy/prisoner-offender-search/values.yaml
   allowlist:
-    prum-test-platform-1: "18.134.149.68/32"
-    prum-test-platform-2: "18.133.201.123/32"
-    prum-test-platform-3: "52.56.76.34/32"
+    prum-test-platform-1: 18.134.149.68/32
+    prum-test-platform-2: 18.133.201.123/32
+    prum-test-platform-3: 52.56.76.34/32
+    groups: []
```

Comment:
@andrewrlee this doesn't look right - overwriting the `groups: internal` from the `values.yml` with an empty array?x

Reply:
3 of the 80 odd repos I've raised PRs on rely on merging keys of multiple values files and the script only considers each values file in isolation. 

I'll point fix this but might be worth considering just living with the duplication rather than spreading the definition of what makes up an allowlist across multiple files in the repo and also groups lists.